src/components/totals/TotalDistance/index.js

```diff
@@ -67,19 +85,19 @@ module.exports = React.createClass({
     }
   },
 
-  renderTotal: function() {
-    var symbol     = this.t('symbol');
-    var title      = this.t('title');
-    var totalKms   = this.state.total / 100;
-    var totalMiles = totalKms * 0.621371192;
-    var emptyLabel = this.t('emptyLabel');
-    var formattedTotal;
-
+  formatDistance: function(meters) {
     if (this.props.unit === 'km') {
-      formattedTotal = numeral(totalKms).format(this.props.format);
+      return numeral(meters / 1000).format(this.props.format);
     } else {
-      formattedTotal = numeral(totalMiles).format(this.props.format);
+      return numeral(meters * 0.000621371192).format(this.props.format);
```

Comment:
Can we define this magic number as a constant like `var METERS_TO_MILES = 0.00062...` at the top of the file?